core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1822,10 +1822,17 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_LOST_WORKER_FILE_DETECTION_INTERVAL =
       new Builder(Name.MASTER_LOST_WORKER_FILE_DETECTION_INTERVAL)
-          .setAlias("alluxio.master.worker.heartbeat.interval")
```

Comment:
Assign this alias to `MASTER_LOST_WORKER_DETECTION_INTERVAL` for compatibility?

Reply:
done